src/questions/chapter5.yaml

```diff
@@ -0,0 +1,1075 @@
+  answerKeys:
+    - D
+  topic: Library
+  subTopics:
+    - variables
+    - storage
+  difficultyLevel: Low
+  choices:
+    - content: state variable
+      key: A
+      order: 0
+    - content: fallback
+      key: B
+      order: 1
+    - content: None
+      key: C
+      order: 2
+    - content: Both A and B
+      key: D
+      order: 3
+
+- uuid: d608a41e-befd-4633-b74d-aef833e1f8e6
+  type: SingleChoice
+  content: Type of functions those can be called directily from outside of the library.
+  hint: noHint
+  explanation: Functions of the library can be called directly when they do not modify the state variables i.e. only pure and view functions can be called from outside of the library.
+  answerKeys:
+    - C
+  topic: Library
+  subTopics:
+    - pure
+    - view
+  difficultyLevel: Low
+  choices:
+    - content: pure
+      key: A
+      order: 0
+    - content: view
+      key: B
+      order: 1
+    - content: Both A and B
+      key: C
+      order: 2
+    - content: None
+      key: D
+      order: 3
+
+- uuid: 5112f0d8-19b8-4140-ae06-ae1309acd783
+  type: SingleChoice
+  content: Find the incorrect property about Interface.
+  hint: noHint
+  explanation: You can't put any state variables inside Interfaces, because interfaces define contracts which can be implemented in various ways. T
+  answerKeys:
+    - D
+  topic: Abstraction
+  subTopics:
+    - interface
+  difficultyLevel: Low
+  choices:
+    - content: Interface can not have any function with implementation.
+      key: A
+      order: 0
+    - content: Functions of an interface can be only of type external.
+      key: B
+      order: 1
+    - content: Interface can not have constructor.
+      key: C
+      order: 2
+    - content: Interface can have state variables.
+      key: D
+      order: 3
+
+- uuid: b3466145-78de-407c-b6c3-8dec7756cbb3
+  type: SingleChoice
+  content: Which of following a interface can not have?
+  hint: noHint
+  explanation: Interfaces do not have any definition or any state variables, constructors, or any function with implementation, they only contain function declarations i.e. functions in interfaces do not have any statements.
+  answerKeys:
+    - D
+  topic: Abstraction
+  subTopics:
+    - interface
+  difficultyLevel: Low
+  choices:
+    - content: state variable
+      key: A
+      order: 0
+    - content: constructor
+      key: B
+      order: 1
+    - content: function with implementation
+      key: C
+      order: 2
+    - content: All of these
+      key: D
+      order: 3
+
+- uuid: 3f034ef2-066b-404a-9a3b-dc0992e37d37
+  type: SingleChoice
+  content: Functions of Interface can be only of type 
+  hint: noHint
+  explanation: Functions of Interface can be only of type external.
+  answerKeys:
+    - B
+  topic: Abstraction
+  subTopics:
+    - interface
+    - function
+  difficultyLevel: Low
+  choices:
+    - content: public
+      key: A
+      order: 0
+    - content: external
```

Comment:
Can they not be public?

Reply:
![Screenshot from 2022-08-12 19-54-59](https://user-images.githubusercontent.com/76400354/184374448-2195dc20-b320-40b4-899d-3e2d5851a755.png)

If public they same could happen internally :thinking: